src/drivers/nuopc/cime_driver/esm.F90

```diff
@@ -1583,6 +1583,9 @@ subroutine AddAttributes(gcomp, driver, config, compid, compname, rc)
        if (shr_nuopc_methods_ChkErr(rc,__LINE__,u_FILE_u)) return
 
        if (seq_comm_suffix(compid) /= '') then
+          call NUOPC_CompAttributeAdd(gcomp, attrList=(/'inst_suffix'/), rc=rc)
```

Comment:
@jedwards4b This should resolve the attribute issue - only add attribute if needed.